source/Ship.h

```diff
@@ -596,6 +596,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Links between escorts and parents.
 	std::vector<std::weak_ptr<Ship>> escorts;
 	std::weak_ptr<Ship> parent;
+
+	bool removeBays;
```

Comment:
```suggestion
	bool removeBays = false;
```
We need to initiate this!